### PR TITLE
[docs] updates to wb_to_df(), wb_add_pivot_table() and options()

### PR DESCRIPTION
**Pull request by JanMarvin**

### CODE REVIEW

File: R/class-workbook-wrappers.R [hunk 1]
```diff
@@ -324,7 +366,7 @@ wb_add_data_table <- function(
 #' @param fun A vector of functions to be used with `data`
```
Comment (olivroy) [inline R/class-workbook-wrappers.R:366]:
```suggestion
#' @param fun A vector of functions to be used with `data`. See **Details** for the list of available options.
```

File: R/class-workbook-wrappers.R [hunk 2]
```diff
@@ -324,7 +366,7 @@ wb_add_data_table <- function(
...
 #' @param params A list of parameters to modify pivot table creation.
```
Comment (olivroy) [inline R/class-workbook-wrappers.R:367]:
```suggestion
#' @param params A list of parameters to modify pivot table creation. See **Details** for available options.
```

File: R/class-workbook-wrappers.R [hunk 3]
```diff
@@ -299,19 +299,61 @@ wb_add_data_table <- function(
...
 #' `PRODUCT`, `STDEV`, `STDEVP`, `SUM`, `VAR`, `VARP`.
 #'
 #' `show_data_as` can be any of `normal`, `difference`, `percent`, `percentDiff`,
 #' `runTotal`, `percentOfRow`, `percentOfCol`, `percentOfTotal`, `index`.
 #'
-#' The sheet will be empty unless it is opened in spreadsheet software.
-#'
-#' Find more details in the [section about pivot tables](https://janmarvin.github.io/ox2-book/chapters/openxlsx2_pivot_tables.html)
-#' in the openxlsx2 book.
+#' Possible `param` arguments are listed below. Pivot tables accepts more
```
Comment (olivroy) [inline R/class-workbook-wrappers.R:330]:
```suggestion
#' Possible `params` arguments are listed below. Pivot tables accepts more
```

File: R/class-workbook-wrappers.R [hunk 4]
```diff
@@ -299,19 +299,61 @@ wb_add_data_table <- function(
...
+#' params, but they were either not tested or misbehaved (probably because we
```
Comment (olivroy) [inline R/class-workbook-wrappers.R:331]:
```suggestion
#' parameters, but they were either not tested or misbehaved (probably because we
```

**Review by olivroy — APPROVED** [review]
Great improvement!

Maybe add an example of a pivot table that uses `params` to make it clear how to supply a list?

**Comment by JanMarvin** [comment]
Thanks, for the review! I’ll merge your suggestions. I want to add something for slicers too and will add an example as suggested!

**Pull request merged**